src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
```

Comment:
I don't have a strong opinion. I sometimes wonder whether C# or C++ start enum values at 0 or 1 by default, and whether the two match on that behavior. Sometimes that question causes me to explicitly specify the values just to make sure they match. But if they match naturally (and I suppose they probably do) then I suppose as long as folks don't mind me asking in PRs whether the values are assigned in native code and we're missing them here, I guess we don't need to require it when implicit assignment is enough.